data/gdshowsdb/1971.yaml

```diff
@@ -1295,7 +1295,89 @@
   :city: San Francisco
   :state: CA
   :country: US
-  :sets: []
+  :sets:
+  - :uuid: ea054350-e65d-4e4a-b99c-9f42b111ea10
+    :songs:
+    - :uuid: c0246dee-5fae-4ad6-814d-a718d47371e8
+      :name: Bertha
+      :segued: false
+    - :uuid: 73e14dd3-27be-4010-9094-b88d1ac11d60
+      :name: Me And My Uncle
+      :segued: false
+    - :uuid: 43e4cb07-6698-49ea-bf32-2d5e2e7c847c
+      :name: Sugaree
+      :segued: false
+    - :uuid: 5727375f-3d50-479b-8dce-1c391fec3ef3
+      :name: Jack Straw
+      :segued: false
+    - :uuid: 4abb209c-d53f-4cff-914b-3b494bbd42b7
+      :name: Tennessee Jed
+      :segued: false
+    - :uuid: c77efe2c-7f2f-4168-acb6-18d0f40913cb
+      :name: El Paso
+      :segued: false
+    - :uuid: d99f15ba-7d69-41c5-9ec0-5e1fe973ab4a
+      :name: Loser
+      :segued: false
+    - :uuid: c908bcba-b39f-4e16-98e5-1b2f9f920ca0
```

Comment:
Playing in the Band should be here according to the archive and deadlists.com

Reply:
Fixing now.